src/components/WindowSettings.vue

```diff
@@ -49,5 +49,9 @@ const showQR = ref(false);
   justify-content: center;
   align-items: center;
   gap: 10px;
+  border-style: solid;
+  border-radius: 2.5%;
```

Comment:
```suggestion
  border-radius: 30px;
```
BaseWidgetに準拠するならこっち